main.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+min_random_number_limit = 5
+max_random_number_limit = 19
```

Comment:
```suggestion
MIN_NUM = 5
MAX_NUM = 19
```

Constants are UPPER_CASE :)